src/main/java/ee/tuleva/onboarding/aml/AmlAutoChecker.java

```diff
@@ -45,14 +44,12 @@ public void beforeLogin(BeforeTokenGrantedEvent event) {
   @EventListener
   public void afterLogin(AfterTokenGrantedEvent event) {
     Person person = event.getPerson();
-    String token = event.getAccessToken().getValue();
 
     userService
         .findByPersonalCode(person.getPersonalCode())
         .ifPresent(
             user -> {
-              ContactDetails contactDetails =
-                  contactDetailsService.getContactDetails(person, token);
+              ContactDetails contactDetails = contactDetailsService.getContactDetails(person);
```

Comment:
At the time when AfterTokenGrantedEvent was fired, there was no security context set up yet. That's why we needed to manually pass the token to epis. Is it now different?